plugins/inlinetoolbar/plugin.js

```diff
@@ -18,18 +18,30 @@
 	 * @param {Object} definition An object containing the panel definition.
 	 */
 	CKEDITOR.ui.inlineToolbarView = function( editor, definition ) {
-		CKEDITOR.ui.balloonPanel.call( this, editor, definition );
+		var defParams = CKEDITOR.tools.extend( definition || {}, {
```

Comment:
Instead of adding a new var could you just overwrite the `definition` parameter?